internal/runner/fixtures/rspec/spec/with_skipped_examples_spec.rb

```diff
@@ -0,0 +1,17 @@
+RSpec.describe("Spec with skipped examples") do
+  it("not skipped") do
+    true
+  end
+
+  xit("skiped using xit") do
+    fail
+  end
+
+  it("skipped using skip option", skip: "skipped") do
+    fail
+  end
+
+  it("skipped using pending option", pending: "skipped") do
```

Comment:
Test example marked as `pending` (as opposed to `skip`) will still be executed by RSpec. So we should find a way to include `pending` examples, but exclude `skip` examples.